eql/__init__.py

```diff
@@ -66,7 +66,7 @@
     Walker,
 )
 
-__version__ = '0.9.10'
+__version__ = "0.9.11"
```

Comment:
This broke the version regex parsing from setup.py

Reply:
oh yeah
```suggestion
__version__ = '0.9.11'
```